auth-service/src/main/java/org/libertybikes/auth/service/GoogleAuth.java

```diff
@@ -0,0 +1,71 @@
+/**
+ *
+ */
+package org.libertybikes.auth.service;
+
+import java.net.URI;
+import java.util.Arrays;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+@Path("/GoogleAuth")
+@ApplicationScoped
+public class GoogleAuth {
+    private static final long serialVersionUID = 1L;
+
+    @Inject
+    @ConfigProperty(name = "googleOAuthConsumerKey")
+    String key;
+
+    @Inject
+    @ConfigProperty(name = "googleOAuthConsumerSecret")
+    String secret;
+
+    @Inject
+    @ConfigProperty(name = "auth_url", defaultValue = "https://localhost:8482/auth-service")
+    String authUrl;
+
+    @Context
+    HttpServletRequest request;
+
+    @GET
+    public Response getAuthURL() {
+
+        JsonFactory jsonFactory = new JacksonFactory();
+        HttpTransport httpTransport = new NetHttpTransport();
+
+        GoogleAuthorizationCodeFlow flow = new GoogleAuthorizationCodeFlow(httpTransport, jsonFactory, key, secret, Arrays
+                        .asList("https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"));
+
+        try {
+            // google will tell the users browser to go to this address once
+            // they are done authing.
+            String callbackURL = authUrl + "/GoogleCallback";
+            request.getSession().setAttribute("google", flow);
+
+            String authorizationUrl = flow.newAuthorizationUrl().setRedirectUri(callbackURL).build();
+
+            // send the user to google to be authenticated.
+            return Response.temporaryRedirect(new URI(authorizationUrl)).build();
+
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+        return null;
```

Comment:
instead of returning null here lets return a proper HTTP error code (I think 500) in the catch block.  Also, lets dump the full stack trace (using `e.printStackTrace()`) instead of just printing `e.getMessage()`